src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -347,6 +347,11 @@ public static class ShaderOptions {
             @Config.Name("Fusion Reactor")
             public FusionBloom fusionBloom = new FusionBloom();
 
+            @Config.Comment("Particle config option for the Assembly Line")
+            @Config.Name("Assembly Line Particles")
```

Comment:
Config.name exists for custom class contains multiple configs I guess
You don't have to set this for assembly line particles since its name is already pretty obvious
(Also it looks weird)
![キャプチャ1682](https://user-images.githubusercontent.com/39122497/190885120-0a76bf08-982a-4c94-8196-883f5b6e1822.PNG)

Reply:
The `name` display is mostly for in game, as it titles the buttons and config options in game. However, I have removed this name because we do not name the individual config fields, just the categories.